Assets/HoloToolkit/Input/Tests/Scripts/SphereGlobalKeywords.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule.Tests
+{
+    public class SphereGlobalKeywords : MonoBehaviour, ISpeechHandler
+    {
+        private void Start()
+        {
+            InputManager.Instance.AddGlobalListener(gameObject);
+        }
+
+        public void OnSpeechKeywordRecognized(SpeechKeywordRecognizedEventData eventData)
+        {
+            switch (eventData.RecognizedText.ToLower())
+            {
+                case "reset all":
+                    foreach (Renderer renderer in GetComponentsInChildren<Renderer>())
+                    { 
+                        renderer.material.color = Color.gray;
```

Comment:
Fixed this using Resources.UnloadAsset(). I think that's OK for a test scene and for something that only happens on a voice command.